test/component/deceased/testAnyDeceasedChildren.js

```diff
@@ -37,9 +37,9 @@ describe('any-deceased-children', () => {
         it('test content loaded on the page', (done) => {
             const sessionData = {
                 deceased: {
-                    firstName: 'John',
-                    lastName: 'Doe',
-                    dod_formattedDate: '13 October 2018'
+                    'firstName': 'John',
```

Comment:
these dont need to be wrapped in `'`

Reply:
I know, just for consistency with the other properties in the same object (`dod-formattedDate` does require being wrapped in quotes).